src/assets/images.js

```diff
@@ -3,7 +3,8 @@ const images = {
     gear: require('./images/gear.png'),
     magGlass: require('./images/magnifying-glass.png'),
     recycleBook: require('./images/recycle-book.png'),
-    recycleSymbol: require('./images/recycle-symbol.png')
+    recycleSymbol: require('./images/recycle-symbol.png'), //Robert: Can we have a recycle symbol green?
```

Comment:
- comments like this don't belong in the code
- I want to keep the recycle symbol a different color from the compost and trash symbols